index.html

```diff
@@ -70,7 +70,7 @@ <h2 id="abstract"><a class="toc-anchor" href="#abstract">§</a> Abstract</h2>
 use to submit proofs in accordance with them. The specification is designed to
 be both credential format and transport envelope agnostic, meaning an implementer
 can use JWTs, VCs, JWT-VCs, or any other credential format, and convey them
```

Comment:
why no VPs?

Reply:
Not sure - is that an actual credential handling concern? Is the VP the actual credential thing plucked via the JSONPath expression, or just a random wrapper that is inconsequential in terms of the selection and processing steps of PE?

Reply:
If you support VCs, is there any reason to believe they would not have support for the same signature stuff for VPs?

Reply:
A VP has a signature from the presenter (cred holder) in addition to whatever sig is on the credential. The signature on the presentation could be different than the creds.

> If you support VCs, is there any reason to believe they would not have support for the same signature stuff for VPs?

To give an example, I could have a number of credentials issued against `Ed25519Signature2018` but my wallet supports signing with `JCSEd25519Signature2020`, so the presentation may have a different signature/supported suite than the creds

Reply:
@glcohen I added `vp` and `jwt-vp` to the supported formats list